rocrand/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: Jakub Okoński <jakub@okonski.org>
 pkgname=rocrand
 pkgver=3.5.0
-pkgrel=1
+pkgrel=3
```

Comment:
The pkgrel should only be bumped up by 1. I'll be merging this PR as a single commit.